Orange/data/table.py

```diff
@@ -1065,10 +1065,20 @@ def has_missing(self):
         missing_x = not sp.issparse(self.X) and bn.anynan(self.X)   # do not check for sparse X
         return missing_x or bn.anynan(self._Y)
 
+    def has_missing_attribute(self):
+        """Return `True` if there are any missing attribute values."""
+        return not sp.issparse(self.X) and bn.anynan(self.X)    # do not check for sparse X
+
     def has_missing_class(self):
         """Return `True` if there are any missing class values."""
         return bn.anynan(self._Y)
 
+    def get_nan_percentile_attribute(self):
+        return np.isnan(self.X).sum() / self.X.size * 100
+
+    def get_nan_percentile_class(self):
+        return np.isnan(self._Y).sum() / self._Y.size * 100
```

Comment:
I think it would be better if these two functions reported relative frequencies, not percentages. That is, I would remove `* 100`. Percentages are more "human-friendly", while these two functions look general and could also be used in other, non-GUI-related code.